### ISSUE TITLE
wrong official language code and language name for Spain

### ISSUE THREAD

**Issue by oliwoli** [LucianoGanga/country-codes-list#32]
The official language for Spain is Spanish `es`.
It's set to `ast` at the moment, which I didn't find much information on. Is it Asturian?

Here's the corrected entry:
```
  {
    "countryNameEn": "Spain",
    "countryNameLocal": "España",
    "countryCode": "ES",
    "currencyCode": "EUR",
    "currencyNameEn": "Euro",
    "tinType": "NIF (CIF)",
    "tinName": "Número de Identificación Fiscal (formerly named Código de Identificación Fiscal)",
    "officialLanguageCode": "es",
    "officialLanguageNameEn": "Spanish, Castilian",
    "officialLanguageNameLocal": "Español",
    "countryCallingCode": "34",
    "areaCodes": [],
    "region": "Europe",
    "flag": "🇪🇸"
  },
```

**Comment by gitahievans** [comment]
Can I go ahead and work on this issue?

**Comment by LucianoPaci** [comment]
@gitahievans Sorry, I didn't see your comment. 
This was fixed on v1.6.11!

**Comment by gitahievans** [comment]
No worries @LucianoPaci
